libraries/AP_Common/AP_FWVersionDefine.h

```diff
@@ -21,12 +21,27 @@
 #endif
 
 #include <AP_Common/AP_FWVersion.h>
+#include <AP_Vehicle/AP_Vehicle_Type.h>
 
 const AP_FWVersion AP_FWVersion::fwver{
+    // Version header struct
+    .header = 0x61706677766572fb, // First 7 MSBs: "apfwver", LSB is the checksum of the previous string: 0xfb
```

Comment:
need to check that this header does only appear once in the file (ie. not in a constructor too)

Reply:
From my script and using `hx binary | less` search, the sequence only appears once.